src/Integration.Vsix/Commands/ConnectedModeMenu/ManageConnectionsCommand.cs

```diff
@@ -22,10 +22,12 @@
 using Microsoft.VisualStudio.Shell;
 using SonarLint.VisualStudio.Integration.TeamExplorer;
 
-namespace SonarLint.VisualStudio.Integration.Vsix
+namespace SonarLint.VisualStudio.Integration.Vsix.Commands.ConnectedModeMenu
 {
     internal class ManageConnectionsCommand : VsCommandBase
     {
+        internal const int Id = 0x102;
```

Comment:
I think it makes more sense to just call it Id, this way the usage is `ManageConnectionsCommand.Id` and not `ManageConnectionsCommand.ManageConnectionsCommandId`

Reply:
It did it the same name, so +1